### ISSUE TITLE
WARNING: BLAS library not found.

### ISSUE THREAD

**Issue by SerkanGuldal** [sunqm/pyscf#172]
Hello,
I am having difficult time to install pyscf. I am receiving following error. I hope somebody can help me.

```
[asa@localhost pyscf-master]$ python -m pip install -e .
Obtaining file:///home/asa/Downloads/pyscf-master
    Complete output from command python setup.py egg_info:
    ****************************************************************
    *** WARNING: BLAS library not found.
    * You can include the BLAS library in the global environment LDFLAGS, eg
    *   export LDFLAGS='-L/path/to/blas/lib -lblas'
    * or specify the BLAS library path in  PYSCF_INC_DIR
    *   export PYSCF_INC_DIR=/path/to/blas/lib:/path/to/other/lib
    ****************************************************************
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/asa/Downloads/pyscf-master/setup.py", line 194, in <module>
        raise RuntimeError
    RuntimeError
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /home/asa/Downloads/pyscf-master/
You are using pip version 9.0.1, however version 9.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by sunqm** [comment]
You can set the environment variable PYSCF_INC_DIR to include the BLAS library path.

Your system looks like linux system. I guess you can use another solution which is simpler.  You can download the .whl file from pypi.python.org then unzip the wheel file. Don't forget to add the pyscf directory in your PYTHONPATH environment.

**Comment by SerkanGuldal** [comment]
I have installed but I don't know how it happened. Also Psi4 is a pleasing alternative.

**Comment by sunqm** [comment]
pip wheels were available since pyscf-1.4.4. Now the installation requires just a line of command
```
pip install pyscf
```
or 
```
pip install --upgrade pyscf
```
